src/cowboy_websocket.erl

```diff
@@ -374,10 +391,11 @@ active(State) ->
 	maybe_read_body(State),
 	State#state{active=true}.
 
-passive(State=#state{transport=undefined}) ->
+passive(State=#state{transport={data_delivery, _}}) ->
 	%% Unfortunately we cannot currently cancel read_body.
 	%% But that's OK, we will just stop reading the body
 	%% after the next message.
+	%% @todo We can't stop relay data_delivery currently.
```

Comment:
Stop of reads comes from:

* https://github.com/ninenines/cowboy/issues/1054
* https://github.com/rabbitmq/rabbitmq-web-stomp/issues/40

But to make it work with this feature we need a way to signal that we don't want to continue reading data from the stream.